### PR TITLE
Raise FileNotFoundError when unable to locate the .cfg file specified by --config_file 

### LINKED ISSUE #1038: Raise error when --config_file does not find a valid config file
Using the --config_file flag is really nice to customize manim scripts, e.g. with:
--config_file 'home/manim_settings.cfg'
However, if the config file is not found, manim goes currently automatically back to the default config.
Is this a bug or a feature?
In my opinion, it would be ok to raise an error like
`Error: --config_file could not find a valid config file.`
Or if not an error, at least a warning.

### PR DESCRIPTION
**Pull request by Alexis-Lionel**
## Motivation

Raising the error will stop script execution and let the user know that there are problems with the --config_file location instead of reverting back to the default configuration.
## Overview / Explanation for Changes

Fixes #1038, manim will now raise a FileNotFoundError when it can't find the specified config file via --config_file. Previously it reverted back to the default configuration without notifying the user.
## Oneline Summary of Changes

```
- Raise FileNotFoundError when unable to locate the .cfg file specified by --config_file (1070)
```

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [x] Newly added functions/classes are either private or have a docstring
- [x] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by Alexis-Lionel** [comment]
My bad with the formatting for black-check. I've reformatted it, will re-PR in a moment.

**Comment by kolibril13** [comment]
I ll have a look on it this evening.

**Comment by jsonvillanueva** [comment]
Changes look good, I'm not exactly sure why the Ubuntu + Python3.8 workflow failed in the doctests.

**Review by jsonvillanueva — APPROVED** [review]

File: manim/_config/utils.py [hunk 1]
```diff
@@ -737,6 +738,13 @@ def digest_file(self, filename: str) -> "ManimConfig":
         multiple times.
 
         """
+        if not os.path.isfile(filename):
+            raise FileNotFoundError(
+                errno.ENOENT,
```
Comment (naveen521kk) [inline manim/_config/utils.py:743]: Is it required to add `errno.ENOENT`?

Comment (Alexis-Lionel) [reply manim/_config/utils.py:743]: @naveen521kk It feels like a proper way to do it, but hardcoding this particular error number (2) will likely work just as well.

**Review by naveen521kk — APPROVED** [review]

**Review by kolibril13 — APPROVED** [review]
Tested on mac, works fine!

**Pull request merged**